### PR TITLE
remove support for `crypto.pseudoRandomBytes` in node 11+

### LINKED ISSUE #137: Broken with Node.js 11
This package breaks on node 11 due to `crypto.pseudoRandomBytes` being read only
https://github.com/nodejs/node/commit/221df2286d079a741e844ef50da3e945a1dccb69
https://github.com/nodejs/node/commit/709b3b1e1c75afe6b5dd9d7859d0142dd4d20e5c

(also some tests are failling on node 10). I will PR the removal of support for `crypto.pseudoRandomBytes`

### PR DESCRIPTION
**Pull request by vdeturckheim**
fixes #137

### CODE REVIEW

**Comment by watson** [comment]
Btw, the work I did to try and fix this previously (but put off because of the Node.js 11 test failures), @Qard and I instead implemented in Elastic APM here: https://github.com/elastic/apm-agent-nodejs/commit/68c5524cfcf5e98a5dce2238d0198ad3b8005c81

**Comment by vdeturckheim** [comment]
@watson thanks for the review, I just updated this part. I agree with you but if possible I'd rather have a quick publish of this update as this is currently probably blocking large chunks of the instrumentation ecosystem on node 11.

File: index.js [hunk 1]
```diff
@@ -329,15 +330,16 @@ if (zlib && zlib.Deflate && zlib.Deflate.prototype) {
...
+
+  const toWrap = [
       'pbkdf2',
       'randomBytes',
-      'pseudoRandomBytes',
-    ],
-    activator
-  );
+  ];
+  if (v11plus) {
```
Comment (watson) [inline index.js]:
```suggestion
  if (!v11plus) {
```

**Comment by watson** [comment]
I'm ok with merging without adding v11 tests to get this working quickly

**Comment by vdeturckheim** [comment]
Thanks :D (btw I love suggested changes)

**Comment by vdeturckheim** [comment]
Shame on me, I have not used node < 4 in years

**Comment by watson** [comment]
Released in v0.6.10

**Pull request merged**